hack/python-sdk/swagger.json

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
what generates this file and any idea what purpose it serves?

Reply:
It is generated by `hack/swagger/main.go`. It serves as input to openapi-generator-cli.jar which generates the Python SDK.